### PR TITLE
docs - updates for gphdfs jar file changes

### PR DESCRIPTION
**Pull request by lisakowen**
the recent changes to get gphdfs to build consolidated/removed some jar files.  doc updates related to this work include:
- needed to update a few references to jar file names
- removed notes related to gpmr-1.0 connector since this is not supported
- noticed that the list of supported gp_hadoop_target_version values on https://gpdb.docs.pivotal.io/520/admin_guide/external/g-one-time-hdfs-protocol-installation.html differed from the list on the guc reference page (https://gpdb.docs.pivotal.io/520/ref_guide/config_params/guc-list.html#gp_hadoop_target_version)
   - removed some values from the guc reference page
   - did not update the default gp_hadoop_target_version, since it is still gphd-1.1 in the code

this should be backported to 5X_STABLE.

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -4173,7 +4173,7 @@
           <tbody>
             <row>
               <entry colname="col1"
-                  >gphd-1.0<p>gphd-1.1</p><p>gphd-1.2</p><p>gphd-2.0</p><p>gphd-3.0</p><p>gpmr-1.0</p><p>gpmr-1.2</p><p>hadoop2</p><p>hdp2</p><p>cdh5</p><p>cdh3u2</p><p>cdh4.1</p></entry>
+                  >gpmr-1.2<p>hadoop2</p><p>hdp2</p><p>cdh5</p><p>cdh4.1</p></entry>
               <entry colname="col2">gphd-1.1</entry>
```
Comment (mkiyama) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: The default value gphd-1.1 is not part of the allowed values.

Comment (lisakowen) [reply gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: as i mentioned in the description, the default value in the code is still gphd-1.1.

Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: It seems kind of weird that the default is not an allowed value.

**Comment by lisakowen** [comment]
@lavjain - fyi ... these changes are for your first gphdfs PR, which was tagged with 5.3.0.  i will submit another PR for doc changes related to your currently open gphdfs PR.

(i needed to make some gphdfs doc updates in this PR that should have been made for 5.0.0 release.)

**Comment by lisakowen** [comment]
@dyozie -  please review again.

links to pages w/ most changes:
- http://docs-gpdb-review-staging.cfapps.io/530/admin_guide/external/g-one-time-hdfs-protocol-installation.html
- http://docs-gpdb-review-staging.cfapps.io/530/admin_guide/external/g-hdfs-readable-and-writable-external-table-examples.html
- http://docs-gpdb-review-staging.cfapps.io/530/admin_guide/external/g-reading-and-writing-custom-formatted-hdfs-data.html
- http://docs-gpdb-review-staging.cfapps.io/530/ref_guide/config_params/guc-list.html#gp_hadoop_home

**Comment by lisakowen** [comment]
thanks for reviewing, @lavjain !

**Pull request merged**